docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+## Browser List
+
+### Local Browsers
+
+Specify the list of browsers (separated by commas) where tests will be ran.
+You can specify browser paths or [aliases](/testcafe/documentation/using-testcafe/common-concepts/browser-aliases). 
+A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
+
+The following example demonstrates how to run a test in several browsers. The browsers can be specified by aliases as well as by paths.
+
+```sh
+testcafe chrome,ff,/applications/safari.app /tests/sample-fixture.js
+```
+
+### All Browsers
+
+To run tests against all installed browsers, use the `all` alias.
+
+```sh
+testcafe all /tests/sample-fixture.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify `remote` as a browser alias. 
+
+```sh
+testcafe remote /tests/sample-fixture.js
+```
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote /tests/sample-fixture.js
+```
+
+TestCafe will provide URLs that you need to open in required browsers on your remote device. 
+
+You can also use the [--qr-code](#qr-code) option to get QR-codes that represent the same URLs. 
+Scan the QR-codes by a device on which you are going to test the application.
+As a result, the browsers will be connected to TestCafe and tests will start.
+
+## File Path/Glob Pattern
```

Comment:
See the up-most comment about the modifier for `file-or-glob`
It all started with me looking at this title at thinking like it doesn't say a lot.
So, I'd change it accordingly